best_practices/controllers.rst

```diff
@@ -209,8 +209,7 @@ flexible::
     }
 
 The point is this: the ParamConverter shortcut is great for simple situations.
-But you shouldn't forget that querying for entities directly is still very
-easy.
+But you shouldn't forget that querying for entities directly is not to hard.
```

Comment:
I don't like my wording here, any improvements?

Reply:
not to difficult?